### PR TITLE
Use user pip conf in docker build

### PR DESCRIPTION
**Pull request by aviem-naturalint**

### CODE REVIEW

File: rainbow/build/image/python/python.py [hunk 1]
```diff
@@ -18,10 +18,14 @@
 
 import os
 
-from rainbow.build.image.image_builder import ImageBuilder
+from rainbow.build.python import BasePythonImageBuilder
 
 
-class PythonImageBuilder(ImageBuilder):
+def __f():
```
Comment (amihayz) [inline rainbow/build/image/python/python.py:24]: What does __f(): return one mean?

File: tests/runners/airflow/rainbow/rainbow.yml [hunk 2]
```diff
@@ -50,7 +50,7 @@ pipelines:
         input_path: '[ { "foo": "bar" }, { "foo": "baz" } ]'
         split_input: True
         executors: 2
-        cmd: python -u helloworld.py
+        cmd: python -u hello_world.py
```
Comment (amihayz) [inline tests/runners/airflow/rainbow/rainbow.yml:53]: is hello_world.py file supposed to be part of this project?

File: rainbow/build/image/python/python.py (hunk 1 above)
Comment (aviem-naturalint) [reply rainbow/build/image/python/python.py:24]: Shouldn't be there, will delete it

File: tests/runners/airflow/rainbow/rainbow.yml (hunk 2 above)
Comment (aviem-naturalint) [reply tests/runners/airflow/rainbow/rainbow.yml:53]: It is, in the tests

**Pull request merged**